gitbutler-ui/src/lib/components/FileContextMenu.svelte

```diff
@@ -29,13 +33,57 @@
 					<ContextMenuItem label="Discard file (Binary files not yet supported)" disabled />
 				{:else}
 					<ContextMenuItem
-						label="Discard file"
+						label={item.files.length === 1 ? 'Discard file' : 'Discard files'}
 						on:click={() => {
 							confirmationModal.show(item);
 							dismiss();
 						}}
 					/>
 				{/if}
+				{#if item.files.length === 1}
+					<ContextMenuItem
+						label="Copy Path"
+						on:click={async () => {
+							try {
+								if (!project) {
+									return;
+								}
+								const absPath = await join(project.path, item.files[0].path);
+								navigator.clipboard.writeText(absPath);
+								dismiss();
+							} catch (err) {
+								console.log('Failed to copy path', err);
+							}
+						}}
+					/>
+					<ContextMenuItem
+						label="Copy Relative Path"
+						on:click={() => {
+							try {
+								if (!project) {
+									return;
+								}
```

Comment:
For consistency with the code base I'd recommend expressing this as: `if(!project) return;`